### DISCUSSION TITLE
Is there options for show selected element as a column, not in a row?

### DISCUSSION THREAD

**Discussion by andrconstruction** [JedWatson/react-select#5852] [Q&A] [answered]
Is there options for show selected elements ( multiple ) as a column, not in a row?

**Comment by manjushsh** [comment, marked as answer]
@andrconstruction , From what I understand, you want to show options after other vertically in multi select.

[Example for the same on Codesandbox (Note: Third select from top)](https://codesandbox.io/p/devbox/affectionate-wood-wrt236?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clp9bsddm00072e66nkk9xaa0%2522%252C%2522sizes%2522%253A%255B73.79746835443038%252C26.20253164556962%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clp9bsddm00032e667nysjcxm%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clp9bsddm00042e668is4ympt%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clp9bsddm00062e66j516i034%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B60.199712110418815%252C39.800287889581185%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clp9bsddm00032e667nysjcxm%2522%253A%257B%2522id%2522%253A%2522clp9bsddm00032e667nysjcxm%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqgf8l2t00022e67vx7wjp3k%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A54%252C%2522startColumn%2522%253A11%252C%2522endLineNumber%2522%253A54%252C%2522endColumn%2522%253A11%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FApp.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clqm8p8oh00022e676pjnbug7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A42%252C%2522startColumn%2522%253A11%252C%2522endLineNumber%2522%253A42%252C%2522endColumn%2522%253A11%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FConfirmBeforeClear.jsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clqm8p8oh00022e676pjnbug7%2522%257D%252C%2522clp9bsddm00062e66j516i034%2522%253A%257B%2522id%2522%253A%2522clp9bsddm00062e66j516i034%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522port%2522%253A3000%252C%2522taskId%2522%253A%2522start%2522%252C%2522id%2522%253A%2522clp9ceytv002z2e665vab9lxo%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clp9ceytv002z2e665vab9lxo%2522%257D%252C%2522clp9bsddm00042e668is4ympt%2522%253A%257B%2522id%2522%253A%2522clp9bsddm00042e668is4ympt%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clp9chzv500wc2e66ml89uc93%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clqfbonxt0014egf6etge52bv%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522id%2522%253A%2522clqfboz8100632e663vx0skxy%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clqfboz8100632e663vx0skxy%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)
<details> 
<summary>
Code reference 
</summary>

```
multiValue: (styles) => ({
    ...styles,
    backgroundColor: "#fff",
    width: "100%",
  }),
```

</details>

**Reply by andrconstruction** [reply]
thanks
